src/components/structures/HomePage.tsx

```diff
@@ -85,7 +85,7 @@ const UserWelcomeTop = () => {
         </MiniAvatarUploader>
 
         <h1>{ _tDom("Welcome %(name)s", { name: ownProfile.displayName }) }</h1>
-        <h4>{ _tDom("Now, let's help you get started") }</h4>
+        <h2>{ _tDom("Now, let's help you get started") }</h2>
```

Comment:
Why is this being changed?

Reply:
Because it broke an AXE rule, https://dequeuniversity.com/rules/axe/html/4.3/heading-order#:~:text=Best%20Practice%3A%20Start%20the%20Main,2%20heading%20elements%20(%20h2%20).